edmm/ctr.json

```diff
@@ -120,11 +128,12 @@
       "EDDP",
       "EDDC",
       "EDDE"
-    ]
+    ],
+    "cpdlc_login": "EDMO"
   },
   {
     "logon": "EDMM_RDG_CTR",
-    "frequency": "132.555",
+    "frequency": "132.550",
```

Comment:
Why this frequency change? `132.555` is the new 8.33kHz frequency that is used everywhere else for RDG.

Reply:
unintended change, will fix